test/visualization/test_visualization.cpp

```diff
@@ -60,6 +60,36 @@ PointCloud<PointNormal>::Ptr cloud_with_normals1 (new PointCloud<PointNormal>);
 search::KdTree<PointXYZ>::Ptr tree3;
 search::KdTree<PointNormal>::Ptr tree4;
 
+pcl::PointCloud<pcl::PointXYZRGB>::Ptr getPointCloud(int pointNumber) {
+  pcl::PointCloud<pcl::PointXYZRGB>::Ptr cloud(new pcl::PointCloud<pcl::PointXYZRGB>());
+
+  for (int i = 0; i < pointNumber; i++) {
+    pcl::PointXYZRGB pt;
```

Comment:
Use C++11 random (or PCL wrappers) instead. Eg: There's a cloud generator that might help in this situation

Reply:
ohh... the work... :D I'll look into it tomorrow maybe 👍